.travis.yml

```diff
@@ -86,5 +86,6 @@ matrix:
 # Use travis_retry during install to reduce the impact of network failures.
 before_install: travis_retry ./Support/Testing/Travis/before-install.py
 install:        travis_retry ./Support/Testing/Travis/install.py
+before_script:  ./Support/Testing/Travis/before-script.sh
```

Comment:
That's an old comment that doesn't make sense anymore (it pointed to a different line before). GitHub display is le best.